{
  "name": "Error workflow",
  "nodes": [
    {
      "parameters": {},
      "type": "n8n-nodes-base.errorTrigger",
      "typeVersion": 1,
      "position": [
        -48,
        16
      ],
      "id": "d52d721b-552b-4fa0-8636-4c877ac5f46c",
      "name": "Error Trigger"
    },
    {
      "parameters": {
        "authentication": "webhook",
        "content": "=The workflow {{ $json.workflow.name }} failed, with the error message: {{ $json.execution.error.message }}. Last node executed: {{ $json.execution.lastNodeExecuted }}. Check this workflow execution here: {{ $json.execution.url }} My Unique ID: c90f3c6f41104c3c4079318f2f5e1bd7",
        "options": {}
      },
      "type": "n8n-nodes-base.discord",
      "typeVersion": 2,
      "position": [
        160,
        16
      ],
      "id": "f6e84fb9-1422-418d-8f4d-536cdc4cef0a",
      "name": "Discord",
      "webhookId": "acf5032e-a96b-4760-b215-6ff7d3ad5e6e",
      "credentials": {
        "discordWebhookApi": {
          "id": "FYQicpXzoQkaDtPl",
          "name": "Discord Webhook account 2"
        }
      }
    }
  ],
  "pinData": {},
  "connections": {
    "Error Trigger": {
      "main": [
        [
          {
            "node": "Discord",
            "type": "main",
            "index": 0
          }
        ]
      ]
    }
  },
  "active": false,
  "settings": {
    "executionOrder": "v1"
  },
  "versionId": "3b580d40-bb51-4e46-8142-0c0020d2eaef",
  "meta": {
    "instanceId": "89a76364c74e88a271a653861ca67ba1daa0bb43d3a81850f0b5cd13cce94538"
  },
  "id": "wwLRiAV38G6h1zAi",
  "tags": []
}